core/impl/src/main/java/com/blazebit/persistence/impl/function/jsonget/OracleJsonGetFunction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014 - 2020 Blazebit.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.blazebit.persistence.impl.function.jsonget;
+
+import com.blazebit.persistence.impl.util.JpqlFunctionUtil;
+import com.blazebit.persistence.spi.FunctionRenderContext;
+
+/**
+ * @author Moritz Becker
+ * @since 1.5.0
+ */
+public class OracleJsonGetFunction extends AbstractJsonGetFunction {
+
+    @Override
+    protected void render0(FunctionRenderContext context) {
+        Object[] jsonPathElements = new Object[context.getArgumentsSize() - 1];
```

Comment:
Same as for JSON_SET. A utility method that accepts `FunctionRenderContext context, int offset` would be more appropriate.

```
        Object[] jsonPathElements = new Object[context.getArgumentsSize() - offset];
        for (int i = 0; i < jsonPathElements.length; i++) {
            try {
                jsonPathElements[i] = Integer.parseInt(JpqlFunctionUtil.unquoteSingleQuotes(context.getArgument(i + offset)));
            } catch (NumberFormatException e) {
                jsonPathElements[i] = JpqlFunctionUtil.unquoteDoubleQuotes(JpqlFunctionUtil.unquoteSingleQuotes(context.getArgument(i + offset)));
            }
        }
        return jsonPathElements;
```